python/kafka/producer/kafka_producer_config.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+# Strimzi environment variables declaration
+TOPIC = "STRIMZI_TOPIC"
+MESSAGE = "STRIMZI_MESSAGE"
+GROUP_ID = "STRIMZI_GROUP_ID"
+MESSAGE_COUNT = "STRIMZI_MESSAGE_COUNT"
+DELAY = "STRIMZI_DELAY_MS"
+
+# default values for environment variables
+KAFKA_PREFIX = "KAFKA_"
+DEFAULT_BOOTSTRAP_SERVERS_ = "localhost:9092"
+DEFAULT_TOPIC = "my-topic"
+DEFAULT_GROUP_ID = "my-group"
+DEFAULT_VALUE_SERIALIZER = "serializer"
+DEFAULT_MESSAGE_COUNT = 10
+DEFAULT_DELAY = "DELAY_MS"
+DEFAULT_MESSAGE = 'Hello World'
+
+# default values for environmental variables in dictionary
+DEFAULT_PROPERTIES = {
+    'bootstrap_servers': 'localhost:9092'
+}
+
+
+# convert the kafka (strimzi) prefix
+def convertEnvVarToPropertyKey(e):
+    return e[e.index('_') + 1:].lower()
+
+
+class KafkaProducerConfig:
+    def __init__(self, topic, message, group_id, message_count, delay_ms, properties):
+        self.topic = topic
+        self.message = message
+        self.group_id = group_id
+        self.message_count = message_count
+        self.delay = delay_ms
+        self.properties = properties
+
+
+def from_env():
+    topic = get_env(TOPIC, DEFAULT_TOPIC)
+    group_id = get_env(GROUP_ID, DEFAULT_GROUP_ID)
+    message = get_env(MESSAGE, DEFAULT_MESSAGE)
+    delay_ms = get_env(DELAY, DEFAULT_DELAY)
+    message_count = get_env(MESSAGE_COUNT, DEFAULT_MESSAGE_COUNT)
```

Comment:
It seems that `message_count` and `delay` are not used

Reply:
`delay` reintroduced for `time.sleep(config.delay)`